### ISSUE TITLE
block_device_mappings defaults

### ISSUE THREAD

**Issue by Z4ck404** [spotinst/terraform-spotinst-ocean-aws-k8s-vng#15]
**What happened**:

I am not sure this really a bug but I am reporting it any ways: 
All attributes in this var are apparently required, terraform fails if they are not set (to null when not needed). In my case I just need to make sure the root volumes are encrypted with the default ebs kms key. I couldn't just do : 

```
  block_device_mappings = [{
    encrypted             = true
  }]

```

and leave everything else to default ! 
I had to set all : 

```
  block_device_mappings = [{
    volume_size           = local.vngs[count.index].disk_size_gb
    device_name           = "/dev/xvda"
    delete_on_termination = true
    kms_key_id            = data.aws_kms_key.ebs.id
    encrypted             = true
    snapshot_id           = null
    iops                  = null
    volume_type           = "gp2"
    throughput            = null
    no_device             = null
  }]
```
  
**What you expected to happen**:

```
  block_device_mappings = [{
    encrypted             = true
  }]
  
```
And let the spotinst provider handle the defaults.

**Comment by sharadkesarwani** [comment]
@Z4ck404 Fix has been released in version v0.8.0.

**Comment by sharadkesarwani** [comment]
@Z4ck404 Fix has been released in version v0.8.0.
